src/HyperFormula.ts

```diff
@@ -1861,15 +1884,20 @@ export class HyperFormula implements TypedEmitter {
    * //   address: { sheet: 0, col: 0, row: 0 },
    * //   newValue: 0.93524248002062,
    * // }]
-   * const changes = hfInstance.moveCells(source, 1, 1, destination);
+   * const changes = hfInstance.moveCells({start: source, end: source}, destination);
```

Comment:
I prefer spacing for readability 
```suggestion
   * const changes = hfInstance.moveCells({ start: source, end: source }, destination);
```